test/filters/asSelectDirectorList.filter.test.ts

```diff
@@ -2,6 +2,7 @@ import { assert } from 'chai'
 import { generateDirectorDetails } from '../fixtures/companyOfficers.fixtures'
 
 import { asSelectDirectorList, GovUKRadio, GovUKRadioDivider } from 'app/filters/asSelectDirectorList.filter'
+import OfficerType from 'app/models/dto/officerType.enum'
```

Comment:
I would expect to see a new test in this file where it assets that the text reads "I'm not a member of this company" when OfficerType is MEMBER